src/lib/components/FormFeedback.js

```diff
@@ -214,9 +214,24 @@ class DisconnectedFormFeedback extends Component {
           return ["pids." + key, value];
         })
       : [["pids", pids]];
+
+    // eslint-disable-next-line camelcase
+    const custom_fields = errors.custom_fields || {};
```

Comment:
to avoid the next EsLint disable:
```suggestion
    const customFields = errors.custom_fields || {};
```